temporary.py

```diff
@@ -1,3 +1,6 @@
+generate_image_files = False
+if generate_image_files:
+    import os; os.environ['pyspecdata_figures']='latex'
```

Comment:
why?

Reply:
setting this to true will generate image files for posting to slack -- you need to create the `auto_figures` directory inside the current directory first